flatliners/prometheusendpoint.py

```diff
@@ -0,0 +1,32 @@
+import flatliners
+import logging
+import pandas as pd
+
+from .baseflatliner import BaseFlatliner
+
+# Set up logging
+_LOGGER = logging.getLogger(__name__)
+
+class PrometheusEndpoint(BaseFlatliner):
+    """docstring for PrometheusEndpoint."""
+    def __init__(self):
+        super().__init__()
+        _LOGGER.info("Prometheus Endpoint initialized.")
+        # Create a Pandas dataframe here
+        # Shape of the df looks like:
+        # clusterID | timestamp | version | weirdness_score
+        #
```

Comment:
Let's say we expose a gauge, that's only one numerical value at a time. When the endpoint is being scraped by prometheus we're returning the value at that time. So no need to keep historical values. Timestamp can go away.
Just report one weirdness_score and one label for the cluster_version and one for the cluster_id


Ask @hemajv about sample code for creating a gauge.

Reply:
I think if we keep the timestamp, we can expose all the processed data at once and won't need to store anything to the buffer.

Reply:
let's see. I think the gauge interface of the prometheus client library doesnt need a timestamp. You'll just update the gauge instance in the `on_next` call, and you're done